iOSBuildTests.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr --output utr
+chmod +x utr
+EDITOR_LOCATION=
```

Comment:
What is the purpose of this line?

Reply:
for testing purposes you can temporary set the editor path, thus removing the need to reenter it every time you run the tests

Reply:
I mean this variable is assigned twice, empty value here and then hardcoded value below.

Reply:
hardcoded value should have been commented :(